.travis.yml

```diff
@@ -33,7 +33,10 @@ install:
  - cabal sandbox init
 # can't use "cabal install --only-dependencies --enable-tests --enable-benchmarks" due to dep-cycle.
 # must split in two separate 'cabal install's since cabal doesn't update the cabal library before it's needed in 'cabal-version' constraints.
- - cabal install "bytestring >= 0.10.4" 'Cabal == 1.24.*' -j
+ - cabal install "bytestring >= 0.10.4" -j
```

Comment:
Hmm. Maybe we should just install CABALVER=1.24 right away instead of upgrading later.
I can't remember why I made it install an older version (the one shipped with Haskell Platform, iirc) and then upgrade it.

So instead of adding CABALUPGR, does it work to just change CABALVER?

Reply:
Alright, let me see.

Reply:
Nope, errors out https://travis-ci.org/kolmodin/binary/builds/388723151

Reply:
> I can't remember why I made it install an older version (the one shipped with Haskell Platform, iirc) and then upgrade it.

The version of `cabal-install` actually doesn't matter, it's about the version of `Cabal` the lib that is shipped with GHC.